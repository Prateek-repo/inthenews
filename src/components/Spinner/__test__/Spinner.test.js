import { render, screen } from '@testing-library/react';

import {shallow, configure} from 'enzyme'
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import toJson from 'enzyme-to-json';

import Container from '../Spinner';



configure({ adapter: new Adapter() });

describe('<Spinner/>', () => {

  it('SnapShot creation', () => {
    const appSnap = shallow(
      <Container/>
    )
  
    expect(toJson(appSnap)).toMatchSnapshot();

  })
})