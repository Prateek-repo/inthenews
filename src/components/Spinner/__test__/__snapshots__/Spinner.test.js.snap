// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spinner/> SnapShot creation 1`] = `
<div
  className="text-center my-3"
>
  <img
    alt="spinnerLoader"
    src=""
  />
</div>
`;
